src/app/pages/organization/site-areas/site-area/site-area.component.html

```diff
@@ -12,7 +12,7 @@
               <div class="row app-organization">
                 <div class="col-md-5 d-block mt-auto">
                   <div class="rotating-card-container"
-                    [ngStyle]="{ 'pointer-events': canCreateSiteArea || canUpdateSiteArea ? 'all' : 'none'}">
+                    [ngStyle]="{ 'pointer-events': readOnly ? 'all' : 'none'}">
```

Comment:
Is the new check equivalent to the old one? if so, shouldn't it be `readOnly ? 'none' : 'all'` ? Or `!readOnly ? 'all' : 'none'` as it is used below (line 24)?

Reply:
Thanks! That check's for the animation only but I missed the `!` 😅